### PR TITLE
feat(tier4_perception_launch): add parameter to control detection_by_tracker on/off

### PR DESCRIPTION
**Pull request by YoshiRi**
## Description

This PR enables users to turn on/off `detection_by_tracker` node.
User can control node existence with `use_detection_by_tracker` parameter in `perception.launch.xml`.

- when `use_detection_by_tracker` is false
  - `detection_by_tracker` node is off
  -  2nd merger node in `detection.launch.xml` will also passthroughed

image

I am concerning that
**pass-through multiple nodes makes launch.xml coding difficult** and it may harder to understand for readers.

## Tests performed

Tested with logsim and it successfully disabled dbt node.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [ ] I've confirmed the [contribution guidelines].
- [ ] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].
- [ ] The PR has been properly tested.
- [ ] The PR has been reviewed by the code owners.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.
- [ ] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml [hunk 1]
```diff
@@ -329,19 +329,24 @@
...
     <let name="merger/input/objects" value="$(var lidar_detection_model)/validation/objects" if="$(var use_validator)"/>
     <let name="merger/input/objects" value="$(var lidar_detection_model)/objects" unless="$(var use_validator)"/>
+    <let name="detection_end_here" value="$(eval &quot;'$(var use_detection_by_tracker)'=='false' and '$(var use_object_filter)'=='false' &quot;)"/>
+    <let name="merger/output/objects" value="$(var output/objects)" if="$(var detection_end_here)"/>
+    <let name="merger/output/objects" value="$(var lidar_detection_model)_roi_cluster_fusion/objects" unless="$(var detection_end_here)"/>
     <include file="$(find-pkg-share object_merger)/launch/object_association_merger.launch.xml">
       <arg name="input/object0" value="$(var merger/input/objects)"/>
       <arg name="input/object1" value="clustering/camera_lidar_fusion/objects"/>
-      <arg name="output/object" value="$(var lidar_detection_model)_roi_cluster_fusion/objects"/>
+      <arg name="output/object" value=""/>
```
Comment (badai-nguyen) [inline launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml:341]:
```suggestion
      <arg name="output/object" value="$(var merger/output/objects)"/>
```

Comment (badai-nguyen) [reply launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml:341]: I think there is a typo.

Comment (YoshiRi) [reply launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml]: @badai-nguyen Sorry I fixed it.

**Review by badai-nguyen — APPROVED** [review]
LGTM

**Comment by kminoda** [comment]
nits: the title is now `tire4`

**Review by miursh — APPROVED** [review]
LGTM

**Pull request merged**